frontend/src/views/SequenceEditView.vue

```diff
@@ -250,7 +250,27 @@ function updateSlide(slide: ISlide): void {
 }
 
 const currentEditingSlot: Ref<LayoutSlot | null> = ref(null);
-const tabs = ref(['Seite']);
+const allTabs = computed(() => {
+  return [
+    i18n.global.t('slide'),
```

Comment:
the one thing i dont like about this, is that the slots then would have to be named after the vuei18n names.

Reply:
Since it's just an internal ID, i'm not sure if this is too big of an issue

I guess it works both ways, I just think the code is a bit simpler if we localize inside the TabbedContainer, and it's just a minor change there - but your decision, in the end